source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1544,14 +1544,16 @@ export abstract class BaseSubmitActionPeer<TAction extends Adaptive.SubmitAction
             { targetVersion: Adaptive.Versions.v1_3, name: "Automatic", value: "auto" },
             { targetVersion: Adaptive.Versions.v1_3, name: "None", value: "none" }
         ]);
+    static readonly disabledUnlessAssociatedInputsChangeProperty = new BooleanPropertyEditor(Adaptive.Versions.v1_0, "disabledUnlessAssociatedInputsChange", "Only enable when associated inputs change");
```

Comment:
Actions is not udpated after I flipped the settings on ElementProperty
![image](https://user-images.githubusercontent.com/6290692/157720242-ebab15c5-4703-4f14-b408-17d0c278213b.png)

Reply:
Good catch. The checkbox shouldn't appear in the property inspector because you haven't selected version 1.6. This is a designer issue, I'll fix. In the meantime, you can test by setting the target to 1.6.